CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 4.0.0-alpha.1
+
+#### General
+
+* `FEAT`: new loading indicator ([#1719](https://github.com/camunda/camunda-modeler/issues/1719))
+* `FEAT`: new validation logic for deployment diagram ([#1709](https://github.com/camunda/camunda-modeler/issues/1709))
+* `FEAT`: save and restore window size ([#576](https://github.com/camunda/camunda-modeler/issues/576))
+* `FEAT`: apply native keyboard shortcuts in log panel ([#1380](https://github.com/camunda/camunda-modeler/issues/1380))
+* `CHORE`: dialogs to return response instead of button ([`42d0ecc`](https://github.com/camunda/camunda-modeler/commit/42d0eccfae6143f472ec60b0280c7267b7d5a4e5))
+* `CHORE`: remove watermark ([#1731](https://github.com/camunda/camunda-modeler/issues/1731))
+
+#### DMN
+
+* `FEAT`: migrate DMN diagrams to DMN 1.3 ([`7eb32a8`](https://github.com/camunda/camunda-modeler/commit/7eb32a80508c214c63257795b37dc430778871f3))
```

Comment:
* Let's add __support for opening DMN 1.2 and DMN 1.3 diagrams__ to the list of features.
* Let's add __Migrate DMN 1.1 diagrams to DMN 1.3 on diagram open__ to the list of features.